src/components/TimeControl/util.ts

```diff
@@ -501,70 +452,46 @@ export function isLiveGame(time_control) {
     return average_move_time > 0 && average_move_time < 3600;
 }
 
-export function fullDurationString(seconds) {
+export function durationString(seconds: number): string {
     let weeks = Math.floor(seconds / (86400 * 7)); seconds -= weeks * 86400 * 7;
     let days = Math.floor(seconds / 86400); seconds -= days * 86400;
     let hours = Math.floor(seconds / 3600); seconds -= hours * 3600;
     let minutes = Math.floor(seconds / 60); seconds -= minutes * 60;
 
-    function plurality(num, single, plural) {
-        num = Math.round(num);
-        if (num > 0) {
-            if (num === 1) {
-                return num + " " + single;
-            }
-            return num + " " + plural;
+    let coarse_fine_time_template = pgettext("Duration strings using two units (e.g. \"1 week 3 days\", \"2 hours 45 minutes\"). This is localizable because some languages may need to change the order of the coarse and fine times.",
+                                             "{{coarse} {{fine}}");
```

Comment:
coarse is missing a `}`

Reply:
Arg. Fixed, and added tests so I don't do that again!